vector/src/main/res/layout/dialog_phone_number_verification.xml

```diff
@@ -1,10 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
-    android:padding="16dp">
+    android:orientation="vertical">
```

Comment:
No padding for this one?

Reply:
Didn't have a way to view this dialog, I can add padding but I won't be able to test.